paimon-core/src/main/java/org/apache/paimon/utils/TagManager.java

```diff
@@ -144,6 +144,22 @@ public void createTag(
         }
     }
 
+    public void renameTag(String tagName, String newTagName) {
+        try {
+            if (!tagExists(tagName)) {
+                throw new RuntimeException(String.format("tag: %s is not exists", tagName));
+            }
```

Comment:
The specified tag name [%s] does not exist ?

Reply:
done